bukkit/build.gradle

```diff
@@ -48,6 +48,10 @@ shadowJar {
     relocate 'com.rabbitmq', 'me.lucko.luckperms.lib.rabbitmq'
     relocate 'org.apache.commons.pool2', 'me.lucko.luckperms.lib.commonspool2'
     relocate 'ninja.leaping.configurate', 'me.lucko.luckperms.lib.configurate'
+
+    manifest {
+        attributes(["paperweight-mappings-namespace": "mojang"])
+    }
```

Comment:
I think this probably needs to be set on the 'outer' jar as I assume that's where Paper looks, i.e. here: https://github.com/LuckPerms/LuckPerms/blob/master/bukkit/loader/build.gradle

Reply:
Oops, should be fixed now